--- sent
+++ heard
@@ -11,7 +11,7 @@
 than
 1
 200
-gujaratis
+gujarati's
 modi
 has
 managed
@@ -35,8 +35,7 @@
 mass
 murderer
 and
-hindutva
-fanatic
+hinderphathanatic
 with
 a
 title
@@ -44,12 +43,13 @@
 his
 own
 choosing
-vikaas
+star
+vikaiya's
 purush
+star
 or
 development
 man
-for
 the
 first
 families
@@ -91,22 +91,24 @@
 tata
 ambani
 and
-mittal
+meadall
+star
+star
 modi
 had
 not
 gotten
 off
+star
 to
 a
 good
 start
 with
-india’s
+india's
 leading
 business
 figures
-nine
 years
 ago
 in
@@ -117,9 +119,9 @@
 of
 indian
 industry
-cii
-the
-country’s
+csw
+the
+country's
 biggest
 and
 most
@@ -150,7 +152,7 @@
 the
 meeting
 was
-organised
+organized
 after
 a
 special
@@ -221,7 +223,8 @@
 trucks
 were
 set
-afire
+a
+fire
 and
 the
 torching
