--- sent
+++ heard
@@ -1,10 +1,15 @@
+star
+star
 in
 the
 second
 week
 of
 january
-2011
+201
+one
+star
+star
 more
 than
 10
@@ -15,19 +20,20 @@
 countries
 descended
 on
-gandhinagar
+gandhi
+nagar
 for
 the
 fifth
-instalment
+installment
 of
 the
-biannual
+biennial
 summit
 branded
 as
 vibrant
-gujarat
+gourgeat
 a
 marathon
 matchmaking
@@ -42,7 +48,7 @@
 bring
 business
 to
-gujarat
+gosarot
 and
 not
 incidentally
@@ -69,12 +75,12 @@
 all
 over
 the
-businessmen
+businessman
 had
 promised
 investments
 in
-excess
+access
 of
 450
 billion
@@ -94,7 +100,8 @@
 the
 media
 had
-obligingly
+a
+bludgingly
 trumpeted
 both
 the
