--- sent
+++ heard
@@ -1,5 +1,6 @@
-vibrant
-gujarat
+vibe
+and
+gruget
 has
 been
 successfully
@@ -14,7 +15,7 @@
 much
 so
 that
-modi’s
+modi's
 american
 lobbying
 and
@@ -53,7 +54,7 @@
 value
 for
 modi
-can’t
+can't
 be
 measured
 by
@@ -77,7 +78,6 @@
 greater
 than
 60
-percent
 for
 pledges
 made
@@ -97,7 +97,6 @@
 that
 only
 25
-percent
 of
 the
 promised
@@ -166,12 +165,12 @@
 been
 one
 of
-india’s
+india's
 most
 business
 friendly
 and
-industrialised
+industrialized
 states
 into
 a
@@ -185,7 +184,8 @@
 of
 annual
 investment
-inflows
+in
+flows
 into
 singular
 triumphant
@@ -204,7 +204,7 @@
 entrepreneurial
 energy
 of
-gujarat
+goudhurat
 to
 overhaul
 his
